pkg/ls/testdata/includeschema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "@context": "https://layeredschemas.org/ls.json",
+    "@id": "http://example.org/Person/schemaBase",
+    "@type": "Schema",
+    "description": [
+    "This schema defines a Person object with firstName, lastName, and a list of contacts.",
+    "Elements of the contact list are references to a Contact schema."
+    ],
+    "valueType": "https://example.org/Person",
+    "layer": {
+        "@type": "Object",
+        "@id": "http://example.org/Person",
+        "attributes": [
+            {
+                "@id": "http://example.org/Person/firstName",
+                "@type": "Value",
+                "attributeName":"firstName"
+            },
+            {
+                "@id": "http://example.org/Person/lastName",
+                "@type": "Value",
+                "attributeName": "lastName"
+            },
+            {
+                "@id": "http://example.org/Person/contact",
+                "@type": "Array",
+                "attributeName": "contact",
+                "arrayElements": {
+                    "@type": "Reference",
+                    "@id": "http://example.org/Person/contact/items",
```

Comment:
Before you copy children of the schema root node, make sure the node types are compatible: FilterAttributeTypes(nodeType) must be equal to FilterAttributeTypes(schemaRootNode type from the copied schema)

When you copy the schemaRootNode, add the labels of the schemaRootNode to this node.